contrib/fwupd.spec.in

```diff
@@ -106,7 +106,7 @@ Requires: bubblewrap
 Requires: shared-mime-info
 
 Recommends: python3
-Recommends: tpm2-tools tpm2-abrmd
+Recommends: tpm2-tss
```

Comment:
Likewise same comment on the debian packaging.

Reply:
Done, see above.